[fix] 안드로이드 CI 수정

<!-- Bug 용 --> 

## 🐞 현재 상황
>/android 디렉토리에서 ci 실행 시 
Run chmod +x gradlew
  chmod +x gradlew
  shell: /usr/bin/bash -e {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.15-6/x64
    JAVA_HOME_17_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.15-6/x64
chmod: cannot access 'gradlew': No such file or directory
Error: Process completed with exit code 1.

해당 에러 발생

## 😆 변경 사항
> CI 가 제대로 동작되어야 함

## ✅ TO-DO
- [ ] CI 수정
## 🗒️ NOTE

--- a/.github/workflows/android-ci.yml
+++ b/.github/workflows/android-ci.yml
@@ -42,7 +42,7 @@ jobs:
         cache: gradle
 
     - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
+      run: chmod +x ./gradlew
       
     - name: Build with Gradle
       run: ./gradlew build --stacktrace

--- a/.github/workflows/android-ci.yml
+++ b/.github/workflows/android-ci.yml
@@ -71,7 +71,7 @@ jobs:
         cache: gradle
 
     - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
+      run: chmod +x ./gradlew
 
     - name: Run ktlintCheck
       run: ./gradlew ktlintCheck --stacktrace
@@ -100,7 +100,7 @@ jobs:
         cache: gradle
 
     - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
+      run: chmod +x ./gradlew
     
     - name: Run tests
       run: ./gradlew test --stacktrace

